porthtml.html

```diff
@@ -78,15 +78,152 @@ <h6 class="LET"><a href="#">LET’S CONNECT</a></h6>
          
           
         </ul>
+        </div>
+        
         <img src="abc.svg" alt="" width="20" height="20">
 
         <img src="rec.svg" alt="" width="20" height="20">
 
+      
+
+    <div id="modal" class="modal modal--active">
+         
+        
+      <div>
+      <img class="whole-card" src="card2.svg" alt="Work preview for project Uber Navigation">
+      </div>
+      <br>
+      <div>
+      <img class="whole-card" src="card1.svg" alt="Work preview for project Uber Navigation">
+      </div>
+      <br>
+      <div>
+        <img class="whole-card" src="card3.svg" alt="Work preview for project Uber Navigation">
+        </div>
+        <br>
+        <div>
+          <img class="whole-card" src="card4.svg" alt="Work preview for project Uber Navigation">
+          </div>
+        
+        <div class="modal__info">
+          <p class="modal__desc">Lorem ipsum dolor sit amet consectetur adipisicing elit. Veniam architecto sint, quae
+            sit nostrum accusamus magnam blanditiis iusto in ducimus eligendi voluptatibus commodi.</p>
+          <div class="modal__tags-buttons-container">
+            <ul class="modal__tech">
+           
+            </ul>
+            <div class="modal__line"></div>
+            <div class="modal__buttons-container">
+              <a href="#" class="modal__button modal__button--see-live" target="_blank">See live<i
+                  class="modal__icon-button fas fa-external-link-alt"></i></a>
+              <a href="#" class="modal__button modal__button--see-src" target="_blank">See source<i
+                  class="modal__icon-button fab fa-github"></i></a>
+            </div>
+          </div>
+        </div>
+      </div>
+    
+
+    <section class="about-us" id="about">
```

Comment:
- [ ] The `about myself` section should be followed the original website. Kindly fix:-
- The background image & the text color.
- The CSS language image.
- The skills & frameworks design.
- Your website
![fyjy](https://user-images.githubusercontent.com/26767383/153653927-ea1dff7d-3bfb-413e-b0f5-08ba7f5d3d55.PNG)
- Original website 
![sfdsfsd](https://user-images.githubusercontent.com/26767383/153653975-580ab28b-2656-4f6b-8b2a-771397351070.PNG)